pkg/cmd/cli/restore/create.go

```diff
@@ -146,6 +148,9 @@ func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.BoolVarP(&o.Wait, "wait", "w", o.Wait, "Wait for the operation to complete.")
 
 	flags.StringVar(&o.ResourceModifierConfigMap, "resource-modifier-configmap", "", "Reference to the resource modifier configmap that restore will use")
+
+	f = flags.VarPF(&o.WriteSparseFiles, "write-sparse-files", "", "Whether to write sparse files when restoring volumes using restic or kopia")
```

Comment:
`when restoring volumes using restic or kopia` -> `during volume data restore`

Let's not explicitly specify restic/kopia, Velero supports multiple types of uploaders, other uploader may also support this option; moreover, this option doesn't work with restic

Reply:
this option could also work for Restic.

Here I've removed the specific uploaders